index.js

```diff
@@ -1,60 +1,29 @@
-const {app, BrowserWindow, ipcMain, Menu, shell} = require('electron')
 const defaultMenu = require('electron-default-menu')
+const window = require('electron-window')
+const electron = require('electron')
+const Env = require('envobj')
+const path = require('path')
 
-const env = process.env.NODE_ENV
+const delegateEvents = require('./lib/delegate-electron-events')
 
-let win, file, link
+const shell = electron.shell
+const Menu = electron.Menu
+const app = electron.app
```

Comment:
I think the standard style guide doesn't cover destructuring.  I think one level for same-name destructuring is fine in general.

Hint: to work out the nested one, just image you are flipping the LHS and RHS, so it's like grabbing `inputClass = manager.options.inputClass`.  Though expressive, this is far too much mental exercise that reduces readability